packages/common/src/transport/alchemy.ts

```diff
@@ -83,72 +66,46 @@ export function isAlchemyTransport(
- * handles the Bundler and Paymaster RPCs for these chains.
- * ```ts twoslash
- * import { alchemy } from "@alchemy/common";
+ * @example
+ * Using JWT:
+ * ```ts
+ * import { alchemyTransport } from "@alchemy/common";
  *
- * const transport = alchemy({
- *  alchemyConnection: {
- *    apiKey: "your-api-key",
- *  },
- *  nodeRpcUrl: "https://zora.rpc.url",
- * });
+ * const transport = alchemyTransport({ jwt: "your-jwt-token" });
  * ```
  *
- * ### Split Transport support
- * Sometimes, the above configuration is still too restrictive and you want to split specific JSON-RPC methods between different transports. We support
- * this by allowing you to pass in a split transport configuration to make this much simpler.
- * ```ts twoslash
- * import { alchemy } from "@alchemy/common";
+ * @example
+ * Using URL directly:
+ * ```ts
+ * import { alchemyTransport } from "@alchemy/common";
  *
- * const transport = alchemy({
- *  // in this example we want to send all eth_chainId requests to a different RPC
- *  overrides: [{
- *    methods: ["eth_chainId"],
- *    transport: http("https://rpc2.url")
- *  }],
- *  // the fallback is where all other methods will be sent to
- *  fallback: http("https://rpc1.url")
- * });
+ * const transport = alchemyTransport({ url: "https://eth-mainnet.g.alchemy.com/v2/your-key" });
  * ```
  *
- * @param {AlchemyTransportConfig} config The configuration object for the Alchemy transport.
- * @param {number} config.retryDelay Optional The delay between retries, in milliseconds.
- * @param {number} config.retryCount Optional The number of retry attempts.
- * @param {string} [config.alchemyConnection] Optional Alchemy connection configuration (if this is passed in, nodeRpcUrl is required).
- * @param {string} [config.nodeRpcUrl] Optional RPC URL for node (if this is passed in, alchemyConnection is required).
- * @param {Array<{methods: string[], transport: Transport}>} [config.overrides] When provided, fallback is also required as this sets up a split transport
- * @param {Transport} [config.fallback] Optionally required if overrides are provided, this is the fallback transport for the split transport config.
- * @param {string} [config.restConnection] Optionally required if overrides are provided rest connection config for the split transport
- * @param {string} [config.fetchOptions] Optional fetch options for HTTP requests.
- * @param {string} [config.rpcUrl] Optional RPC URL.
- * @param {string} [config.apiKey] Optional API key for Alchemy.
- * @param {string} [config.jwt] Optional JSON Web Token for authorization.
- * @returns {AlchemyTransport} The configured Alchemy transport object.
+ * @param {AlchemyTransportConfig} config - The configuration object for the Alchemy transport
+ * @param {string} [config.apiKey] - API key for Alchemy authentication
+ * @param {string} [config.jwt] - JWT token for authentication
+ * @param {string} [config.url] - Direct URL to Alchemy endpoint
```

Comment:
or a proxy?

Reply:
yes, i just meant maybe it would be nice to mention that is the jsdoc. something like `"Direct URL to Alchemy endpoint or a proxy"`